contents/handbook/engineering/developing-locally.md

```diff
@@ -166,17 +166,17 @@ On Linux you often have separate packages: `postgres` for the tools, `postgres-s
 
 2. Install the latest Node.js 16 (the version used by PostHog in production) with `nvm install 16`. You can start using it in the current shell with `nvm use 16`.
 
-3. Install yarn with `npm install -g yarn@1`.
+3. Install pnpm with `npm install -g pnpm`.
```

Comment:
This will install the latest version, which is probably not an issue, but if we're already using the `packageManager` field in `package.json`, can we leverage `corepack enable` to use that pinned version? Corepack experimental but it does seem convenient 🤔

Reply:
That's looking awesome indeed! I'm going to merge as-is for now and will update after I've properly checked instructions for corepack (didn't get it running quickly).